### DISCUSSION TITLE
Welcome to OpenMoonstone Discussions!

### DISCUSSION THREAD

**Discussion by joetsoi** [joetsoi/moonstone#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Chernoskill** [comment]
I would love to see a remake, or remaster, of this game. I'd wager most fans would be fine with keeping pretty much everything (graphics, gameplay etc.), but fixing bugs and having some configuration options for higher replayability. Things like switching the dragon on or off, number of A.I. knights, starting ear/gold.
I have been starting similar remakes of classic and obscure games since about 2015 in Unity, with the last years spent more on graphics creation.
Anyway, I hope this project will see a full release. If not, I might at least get some inofrmation about how to load the original game's assets.

**Reply by joetsoi** [reply]
Yeah, I've been working on it over christmas, requires and upgrade of some of the libraries I've been using. Plan is to just keep everything exactly the same

**Reply by Chernoskill** [reply]
Good to hear, unfortunately I only have experience with C#. Some weeks ago I used Interprite to double the frames of the knight walking animation by generating and then fixing interpolation frames: https://abload.de/img/knight_interpolatedupetj.gif
